src/NBitcoin/RPC/RawFormatter.cs

```diff
@@ -1,25 +1,27 @@
 ﻿#if !NOJSONNET
 using System;
+using System.IO;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
-using System.IO;
 
 namespace NBitcoin.RPC
 {
     abstract class RawFormatter
     {
+        public Network Network { get; set; }
```

Comment:
Is `Network` set anywhere else? Can we make this `readonly` and/or `protected`?

Reply:
Transaction.cs is setting it